Source/DafnyCore/AST/TopLevelDeclarations.cs

```diff
@@ -328,6 +328,7 @@ abstract public class ModuleDecl : TopLevelDecl {
     return Signature;
   }
   public int Height;
+  public Token RootToken = new Token();
```

Comment:
`Root` sounds like a tree is involved, but IIUC the tokens form a list not a tree. How about `FirstToken` as a name?